### PR TITLE
Missing requests file for Kubernetes AddResourceAction

### PR DESCRIPTION
**Pull request by proman3419**

### CODE REVIEW

File: executor/src/test/java/org/k8loud/executor/action/kubernetes/http/addResource.http [hunk 1]
```diff
@@ -8,10 +8,51 @@ Content-Type: application/json
...
+POST localhost:8080/execute
+Content-Type: application/json
+
+{
+  "collectionName": "kubernetes",
+  "actionName": "AddResourceAction",
+  "params": {
+    "params": {
+      "namespace": "default",
+      "resourceDescription": "            kind: ConfigMap\r\n            metadata:\r\n              creationTimestamp: 2022-02-18T18:52:05Z\r\n              name: game-config\r\n              namespace: default\r\n              uid: b4952dc3-d670-11e5-8cd0-68f728db1985\r\n            data:\r\n              game.properties: |\r\n                enemies=aliens\r\n                lives=3\r\n                enemies.cheat=true\r\n                enemies.cheat.level=noGoodRotten\r\n                secret.code.passphrase=UUDDLRLRBABAS\r\n                secret.code.allowed=true\r\n                secret.code.lives=30\r\n              ui.properties: |\r\n                color.good=purple\r\n                color.bad=yellow\r\n                allow.textmode=true\r\n                how.nice.to.look=fairlyNice"
```
Comment (pgorgolew) [inline executor/src/test/java/org/k8loud/executor/action/kubernetes/http/addResource.http:26]: for now its ok, but can be json format. create story 😄

**Review by pgorgolew — APPROVED** [review]

**Comment by pgorgolew** [comment]
14 commits and 1 chagned 😆

File: executor/src/test/java/org/k8loud/executor/action/kubernetes/http/addResource.http (hunk 1 above)
Comment (proman3419) [reply executor/src/test/java/org/k8loud/executor/action/kubernetes/http/addResource.http:26]: https://github.com/k8loud/themis-executor/issues/77

**Comment by proman3419** [comment]
> 14 commits and 1 file changed 😆

Reusing an old branch :P

**Pull request merged**